lib/engine/game/g_18_zoo/game.rb

```diff
@@ -844,9 +846,66 @@ def routes_subsidy(routes)
           subsidy
         end
 
+        def routes_revenue(routes)
+          value = routes.sum { |route| route.revenue.is_a?(Integer) ? route.revenue : route.revenue[:value] }
+          { type: :revenue, value: value }
+        end
```

Comment:
what's going on here? this is not consistnent with any other game which seems problematic